crates/cargo-contract/src/cmd/extrinsics/events.rs

```diff
@@ -100,26 +100,27 @@ impl DisplayEvents {
 
             let event_data = &mut event.field_bytes();
             let mut unnamed_field_name = 0;
-            for (field, field_ty) in event_fields {
+            for field in event_fields {
```

Comment:
Note that changes here are only to fix compilation because of the updating to `subxt` `0.24`, and unrelated to the bugfix itself.